### PR TITLE
Fix linting error

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
This PR fixes a linting error blocking builds from passing.

## Testing done
N/A

## Screenshots
N/A

## Acceptance criteria
- [x] Fix linting error

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: script/utils.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const runCommand = (cmd, forcedExitCode = null) => {
   });
 };
 
-const runCommandSync = (cmd, forcedExitCode = null) => {
+const runCommandSync = cmd => {
```
Comment (ncksllvn) [inline script/utils.js:15]: A neat trick I learned from someone during code review is you can also prefix with an underscore to have the linter ignore it but retain the variable - `_forcedExitCode`

Comment (kelsonic) [reply script/utils.js:15]: Given that nothing happens with that argument I think it makes sense to omit it? But it kinda depends on what they were planning on doing with this I think 🙂

Comment (kelsonic) [reply script/utils.js:15]: I did not know that btw, very handy!! 🔥

**Review by ncksllvn — APPROVED** [review]
Whoops I forgot to approve!

File: script/utils.js (hunk 1 above)
Comment (ncksllvn) [reply script/utils.js:15]: I think I learned it from @bradl3yC awhile ago!

**Review by mchelen-gov — APPROVED** [review]

**Review by cvalarida — APPROVED** [review]
Huh. I wonder if this sneaked in when we were still flipping on the linting rule to catch unused variables?

Thanks for the fix, at any rate. I don't see any issues with it.

**Comment by bkjohnson** [comment]
Yup - this was before the linting rule fix.

**Pull request merged**